include/accessor.h

```diff
@@ -311,13 +316,20 @@ class accessor<DataT, Dims, Mode, target::device> : public detail::accessor_base
 	DataT* m_device_ptr = nullptr;
 	CELERITY_DETAIL_NO_UNIQUE_ADDRESS id<Dims> m_backing_buffer_offset;
 	CELERITY_DETAIL_NO_UNIQUE_ADDRESS range<Dims> m_backing_buffer_range = detail::zero_range;
+#if CELERITY_ACCESSOR_BOUNDARY_CHECK
+	detail::buffer_id m_accessed_buffer_id; // = nullptr;
+	id<3>* m_attempted_index = nullptr;
```

Comment:
`index` sounds like it's singular, when in reality it's two indices. Maybe call it something like `m_oob_indices`, or consider splitting it into two, e.g. `m_oob_idx_min` and `m_oob_idx_max`.